trace/widgets/elog_post_modal.py

```diff
@@ -0,0 +1,117 @@
+from qtpy.QtGui import QPixmap
+from qtpy.QtWidgets import (
+    QLabel,
+    QDialog,
+    QWidget,
+    QLineEdit,
+    QTextEdit,
+    QListWidget,
+    QMessageBox,
+    QSizePolicy,
+    QVBoxLayout,
+    QDialogButtonBox,
+)
+from services.elog_client import get_logbooks
+
+from widgets.settings_components import SettingsTitle, SettingsRowItem
+
+
+class ElogPostModal(QDialog):
+    def __init__(
+        self,
+        parent: QWidget = None,
+        image_bytes: bytes | None = None,
+    ):
+        super().__init__(parent)
+
+        self.setModal(True)
+
+        main_layout = QVBoxLayout()
+        self.setLayout(main_layout)
+
+        modal_label = SettingsTitle(self, "New Elog Entry", size=14)
+        main_layout.addWidget(modal_label)
+
+        if image_bytes is not None:
+            pixmap = QPixmap()
+            pixmap.loadFromData(image_bytes)
+            image_label = QLabel()
+            image_label.setPixmap(pixmap)
+            image_label.setScaledContents(True)
+            image_label.setFixedSize(400, 300)
+            main_layout.addWidget(image_label)
+
+        self.title_edit = QLineEdit(self)
+        self.title_edit.setSizePolicy(QSizePolicy.Expanding, QSizePolicy.Fixed)
+        self.title_edit.setMinimumWidth(250)
+        title_row = SettingsRowItem(self, "Title\n(required)", self.title_edit)
+        main_layout.addLayout(title_row)
+
+        self.body_edit = QTextEdit(self)
+        body_row = SettingsRowItem(self, "Body\n(optional)", self.body_edit)
+        main_layout.addLayout(body_row)
+
+        self.logbook_list = QListWidget(self)
+        self.logbook_list.setSelectionMode(QListWidget.MultiSelection)
+        logbook_row = SettingsRowItem(self, "Logbooks", self.logbook_list)
```

Comment:
This field is required but doesn't say so like the other fields
```suggestion
        logbook_row = SettingsRowItem(self, "Logbooks\n(required)", self.logbook_list)
```